forc/src/cli/commands/plugins.rs

```diff
@@ -78,7 +78,7 @@ fn format_print_description(
             .to_string()
     };
 
-    let description = parse_description_for_plugin(&display);
+    let description = parse_description_for_plugin(path.to_str().unwrap());
```

Comment:
Tested and works fine. I think we can leave this as is though.

Reply:
@tritao 
- I agree :) 
- I'll defer to @mitchmindtree on this as I'm not sure why there are arbitrary folders in `~/.cargo/bin` or how many there can/might be. But other than that, LGTM

Reply:
@tritao instead of passing in a `&str` here, it looks like we might be better off changing the `parse_description_for_plugin` function to take a `&Path` instead.

We can still call `Command::new(plugin)` directly inside `parse_description_for_plugin` (without any str conversion) as `Path` implements `AsRef<OsStr>`.

We should also update its comment to reflect that it expects the canonical path to the plugin, not just the plugin name.

Reply:
Thanks, that is cleaner for sure, will update the PR